modules/ROOT/pages/index.adoc

```diff
@@ -25,6 +25,44 @@ Installing Runtime Fabric within an existing Kubernetes-based PaaS is not suppor
 
 If you are already using a PaaS solution, MuleSoft recommends deploying Runtime Fabric in parallel with your PaaS. This enables you to take advantage of the complete benefits of Anypoint Platform.
 
+== Shared Responsibility for On-Premises Runtime Fabric
+
+The successful operation of on-premises Runtime Fabric instances is a shared responsibility. It is critical to understand which areas you must manage and which areas are managed by MuleSoft. Without the required infrastructure and proper maintenance, Runtime Fabric will not operate successfully.
```

Comment:
Suggest rewording the last sentence to be more positive in light of RTF.

"Runtime Fabric will only operate successfully if it's provided the required infrastructure and maintenance."

Reply:
Updated this - thank you!